### ISSUE TITLE
How about merging into the Playtika/feign-reactive repo?

### ISSUE THREAD

**Issue by wplong11** [stas29a/coroutine-feign-client#1]
If it is merged to [Playtika/feign-reactive](https://github.com/Playtika/feign-reactive),
it seems that the library can be used with a little more trust.
If the repo is separated, can it not be updated well?
Is there any reason why it shouldn't be merged?

**Comment by stas29a** [comment]
Yes, you are right it should be merged to that library. I did it because I need it right now and can't wait merge)

**Comment by amitojduggal** [comment]
Hello @stas29a  :), thanks for this piece of code, did you manage to create a Merge request to feign-reactive? This is a very useful change.
